### PR TITLE
Use GithubActions instead of TravisCI

### LINKED ISSUE #18: Travis CI PHP 8.2 version workaround
Travis is missing a library when building PHP 8.2 environment. At the moment I implemented a workaround I described here:

https://travis-ci.community/t/php-8-2-support/13401

I would need to come back to this later when Travis properly supports 8.2.

### PR DESCRIPTION
**Pull request by OskarStark**
closes #18

### CODE REVIEW

**Comment by OskarStark** [comment]
I think this will first run after a merge to master. Can you please merge this, I will take care of it in a follow-up PR if sth. isn't working as expected. Thanks

**Comment by slunak** [comment]
Hey, thanks for the PR. I don't think anything will run after this PR. Should the actions be configured in `.actions` folder which in now excluded in `.gitignore`?

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 vendor/
 composer.lock
 .php_cs.cache
-.idea/
\ No newline at end of file
+.idea/
+.github/
```
Comment (OskarStark) [inline .gitignore:5]:
```suggestion
```

**Comment by OskarStark** [comment]
Argh, I thought it was the `.gitattributes` file, my bad. Updated

**Pull request merged**